src/operator/swapaxis-inl.h

```diff
@@ -218,6 +227,11 @@ class SwapAxisProp : public OperatorProperty {
     return {out_grad[swapaxisenum::kOut]};
   };
 
+  std::vector<ResourceRequest> BackwardResource(
+      const std::vector<TShape> &in_shape) const override {
+    return {ResourceRequest::kTempSpace};
+  }
+
   Operator* CreateOperator(Context ctx) const override {
     LOG(FATAL) << "Not Implemented";
     return NULL;
```

Comment:
We should also revise the test (https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_operator.py#L361-L377).